app/assets/stylesheets/grader.scss

```diff
@@ -51,32 +51,32 @@
   width: 100%;
 
   li {
-    border-bottom: 1px solid $light-grey;
+    border-bottom: 1px solid $background-main;
```

Comment:
Won't this make the border blend in with the background?

Reply:
Yes, whoops!